buildpipeline/DotNet-CoreFx-Trusted-Linux-Crossbuild.json

```diff
@@ -416,7 +416,7 @@
       "value": "/crossrootfs/$(PB_Architecture)",
       "allowOverride": true
     },
-    "portableLinux": {
+    "portableBuild": {
```

Comment:
We do not want to error out because this highlevel configuration is also used to generate portable RID packages, and optionally, where applicable, also influence the build (e.g. native build).